pkg/crawler/observable.go

```diff
@@ -56,17 +103,28 @@ func (t *TransactionObservable) observe(
 	explorerSvc explorer.Service,
 	errChan chan error,
 	eventChan chan Event,
+	observableStatus *observableStatus,
```

Comment:
do we actually need a Status? why not simplifying with a bool like `isWaiting`.. we're interested in knowing if the current observation is waiting or not.. we seem to don't care at all about its new or processed states.

Reply:
New would makes sense to eventually drop the same observable if is still waiting (ie. FILO stack)